_includes/ee-linux-install-reuse.md

```diff
@@ -269,19 +279,22 @@ upgrade Docker EE.
     Docker is installed but not started. The `docker` group is created, but no
     users are added to the group.
 
-3.  Edit `/etc/docker/daemon.json`. If it does not yet exist, create it.
-    Assuming that the file was empty, add the following contents.
+{% if linux-dist == "centos" or linux-dist == "rhel" or linux-dist == "oraclelinux" %}
+3.  If you need to use `devicemapper`, edit `/etc/docker/daemon.json`. If it
```

Comment:
Wondering if we should remove the reference to the `daemon.json` file here entirely, because devicemapper on loop is not a supported configuration, so just referring them to the devicemapper guide (as we already do below) should give them all the information they need to setup devicemapper.

The risk is that they make this edit (`"storage-driver": "devicemapper"`), and think they're done (because: look! it works)

Alternatively; with devicemapper now having automatic configuration, perhaps we could inline the minimum steps here, and refer to that document for additional configuration options

(Possibly out of band for this PR)

Reply:
Done